packages/renderer/MdToHtml/rules/checkbox.ts

```diff
@@ -119,6 +119,9 @@ function checkboxPlugin(markdownIt: any, options: RuleOptions) {
 		const tokens = state.tokens;
 		const Token = state.Token;
 
+		// Store the markdown source, and split it line by line.
+		const source = state.src.split('\n');
```

Comment:
This line will be executed for every single note, even those that don't include checkboxes, so we can't really have this here. Is there no way to look at the previous token or the parent token for example? Markdown-it tokens include a lot of information, so I expect what we need is there somewhere.